templates/company-board/README.md

```diff
@@ -2,12 +2,20 @@
 
 ## Usage
 
+Create new tokens for the company-board entity:
+
+```
+template.newTokens()
+```
+
+Create a new company with board entity:
+
 ```
-company.prepareInstance(id)
-company.setupInstance(boardHolders, shareHolders, shareStakes)
+template.prepareInstance(name)
+template.setupInstance(boardHolders, shareHolders, shareStakes)
 ```
 
-- `id`: Identifier to be registered in AragonID
+- `name`: Name for org, will assign `[name].aragonid.eth` (check capitalization)
```

Comment:
I'm fairly confident (not 100%) that ENS names, like domain names, are not case-sensitive.

Reply:
Sure. I brought that comment from the democracy readme and I thought it may be useful advice.

Reply:
From https://github.com/ethereum/EIPs/blob/master/EIPS/eip-137.md#name-syntax:

> Note that while upper and lower case letters are allowed in names, the UTS46 normalisation process case-folds labels before hashing them, so two names with different case but identical spelling will produce the same namehash.

So I think we're safe to remove this comment :)